_includes/post_info.html

```diff
@@ -7,7 +7,9 @@
       <img src="{{ author.avatar | relative_url }}">
     {% endif %}
     <p class="meta">
-      {% if author.name %}{{ author.name }} - {% endif %}{{ date | date: "%B %-d, %Y" }}
+      {% if author.name %}{{ author.name }} - {% endif %}
+      {% assign x = date | date: "%m" | minus: 1 %}
+      {{ site.data.language.str_months[x] }} {{ date | date: "%d, %Y" }}
```

Comment:
I would use:

```
{{ site.data.language.str_months[x] | default: date | date: "%B" }}
```

So that there's a default even if the `language.yml` were to disappear, the month would still appear